frontend/src/js/controllers/dashCtrl.js

```diff
@@ -49,12 +52,14 @@
 
         vm.hostChallenge = function() {
 
-            var alert = $mdDialog.alert()
-                .title('Host a challenge')
-                .htmlContent('Please send an email to <a href="mailto:admin@cloudcv.org" class="blue-text">admin@cloudcv.org</a> with the details of the challenge and we will get back soon.')
-                .ok('Close');
-
-            $mdDialog.show(alert);
+            if ($rootScope.isAuth === true)
```

Comment:
@RishabhJain2018 there is no need to check for auth here as you can't open dashboard until or unless you are logged in. So please modify this accordingly.